### PR TITLE
bump the versions of various toolchain deps

### PR DESCRIPTION
**Pull request by tariq1890**

### CODE REVIEW

File: .golangci.yaml [hunk 1]
```diff
@@ -36,8 +36,6 @@ linters:
     - errchkjson
     - errname
     #- errorlint
-    - execinquery
```
Comment (tariq1890) [inline .golangci.yaml:39]: These linters are no longer relevant with the new golangci-lint version

**Review by adrianchiris — APPROVED** [review]

**Comment by tariq1890** [comment]
Thanks for the approve @adrianchiris ! Merging this PR as it will restore CI pipeline status to green on `main`

**Pull request merged**